packages/suite/src/components/suite/modals/ReviewTransaction/components/OutputList.tsx

```diff
@@ -151,9 +151,17 @@ const OutputList = ({
                             const state = signedTx
                                 ? 'success'
                                 : getState(index, buttonRequests.length);
-                            // it's safe to use array index since outputs do not change
-                            // eslint-disable-next-line react/no-array-index-key
-                            return <Output key={index} {...output} state={state} symbol={symbol} />;
+
+                            return (
+                                <Output
+                                    // it's safe to use array index since outputs do not change
+                                    // eslint-disable-next-line react/no-array-index-key
+                                    key={index}
+                                    {...output}
+                                    state={state}
+                                    symbol={symbol}
+                                />
+                            );
```

Comment:
I would prefer not to reformat parts of code where there are no actual changes. it improves PR readability.